renderdoc/core/target_control.cpp

```diff
@@ -602,6 +623,15 @@ struct TargetControl : public ITargetControl
       SAFE_DELETE(m_Socket);
   }
 
+  void ChangeContext()
+  {
+    WRITE_DATA_SCOPE();
```

Comment:
Similarly to above, this needs to be protected by a version check - if the target program doesn't support the packet it shouldn't be sent. This function would just need to return without doing anything, perhaps printing an error message.

Reply:
I have modified my commit